### PR TITLE
Support for iOS 11, and bug fix

### PR DESCRIPTION
**Pull request by anirudhramanan**
1. Supports iOS 11
2. Bug Fix (UI distortion) when the app comes to foreground

### CODE REVIEW

**Comment by anirudhramanan** [comment]
+ @naveen-c

**Comment by tristanhimmelman** [comment]
Thank you but I already merged another swift 4 branch.

File: HidingNavigationBar/HidingNavigationBarManager.swift [hunk 1]
```diff
@@ -216,6 +219,7 @@ open class HidingNavigationBarManager: NSObject, UIScrollViewDelegate, UIGesture
 		default:
 			break;
 		}
+		handleScrolling()
```
Comment (anirudhramanan) [inline HidingNavigationBar/HidingNavigationBarManager.swift:222]: @tristanhimmelman  This was a bug fix, where the nav bar position gets messed up, when coming back from control center or background, What this taken care of ?

Comment (tristanhimmelman) [reply HidingNavigationBar/HidingNavigationBarManager.swift:222]: I missed this! Sorry about that. I am working to include this and you iOS11 handling of insets.

Comment (anirudhramanan) [reply HidingNavigationBar/HidingNavigationBarManager.swift:222]: Sure, Thanks 👍

**Pull request closed without merging**